includes/application/forwardApplication.php

```diff
@@ -57,7 +57,7 @@
 	$app_id = $_POST['app_id'];
 	$app_type = $_POST['app_type'];
 	$status = $_POST['status'];
-	
+	 
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->